src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/SmeltersTool.java

```diff
@@ -0,0 +1,96 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.tools;
+
+import io.github.thebusybiscuit.slimefun4.api.MinecraftVersion;
```

Comment:
the import order should be the same as other files

Reply:
mb